### PR TITLE
WIP: Send Kubernetes metrics over Helm with the OpenTelemetry Collector

### PR DESCRIPTION
**Pull request by yotamloe**
### What changed

- New integration in the Metrics (Prometheus) tab appears here:  https://deploy-preview-1059--logz-docs.netlify.app/shipping/#prometheus-sources
-  New topic: https://deploy-preview-1059--logz-docs.netlify.app/shipping/prometheus-sources/otel-metrics-k8s-helm.html

New helm chart for Opentelemtry metrics for k8s 
This doc PR relates to the code PR https://github.com/logzio/logzio-helm/pull/34
1. Move topic from snippet directory to correct _metrics-sources folder so it appears in the Send Your Data (SYD) pages.
2. Add logo
3.  Needs logical title. Send Kubernetes metrics over Helm via OpenTelemetry
4. Missing the following metadata

```
data-source: Kubernetes over Helm ???
open-source: ???
  - title: ???
    github-repo: ???
templates: ["docker"]
```
   
Clarify: Where does this belong? :         https://github.com/logzio/logzio-helm/tree/master/charts/opentelemetry

AI: The ReadMe will need to be updated after this document is finalized.

### CODE REVIEW

File: _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Ship Kubernetes metrics over Helm with the OpenTelemetry Collector
+logo:
+  logofile: k8s-helm.svg
+  orientation: horizontal
+data-source: Kubernetes over Helm
+open-source:
+  - title: 
+    github-repo: 
+templates: ["docker"]
```
Comment (YajB) [inline _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md:10]: @Doron-Bargo & @yotamloe 

For lines 6-9 and 15: What should the values be for these metadata fields?

Comment (yotamloe) [reply _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md:10]: This should be the url after we merge: [https://github.com/logzio/logzio-helm/tree/master/charts/opentelmetry](https://github.com/logzio/logzio-helm/tree/master/charts/opentelmetry)

File: _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+##### To uninstall the `logzio-otel-k8s-metrics` deployment
+
+Use the following command:
+
+```shell
+helm uninstall logzio-otel-k8s-metrics
+```
+</div>
+
+### Change log
```
Comment (YajB) [inline _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md:103]: Where is the content for this section? 
How is it related to the previous sections?

Comment (yotamloe) [reply _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md:112]: I think we can remove this section from our docs it is more relevant for the Github repo

File: _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---
+title: Send Kubernetes metrics with Helm and the OpenTelemetry Collector
+logo:
+  logofile: k8s-helm.svg
+  orientation: horizontal
+data-source: Kubernetes over Helm with OpenTelemetry
+open-source:
+  - title: logzio-helm-opentelemetry  #is this correct?
+    github-repo: logzio-otel-k8s-metrics  #is this correct? Should it be https://github.com/logzio/logzio-helm/tree/master/charts/opentelemetry
```
Comment (YajB) [inline _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md:9]: @yotamloe : Please review and update these fields.

Comment (yotamloe) [reply _source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md]: @YajB I have updated the requested fields :)

**Review by YajB — APPROVED** [review]
metadata updated.

**Comment by Doron-Bargo** [comment]
@yotamloe the documentaion is looking good, i added a line to install the helm repo

**Pull request merged**